phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/ChooseRepositoryTypePanel.java

```diff
@@ -74,6 +75,8 @@ public RepositoryType fromString(String string) {
         choiceBoxLabel.setLabelFor(choiceBox);
 
         HBox content = new HBox(choiceBoxLabel, choiceBox);
+        content.setAlignment(Pos.CENTER_LEFT);
```

Comment:
It's not only a question about themeability. 
The important benefit of moving **every** style relevant line of code to css is, that we have everything in a single place, in a single file.
This in turn makes it quite easy to change some/fix some wrong layout without the need to search the whole codebase first.

In addition:
I don't think it's bad if make the dialogs themeable. It doesn't give us any disadvantage I think, right?